python/sglang/srt/managers/scheduler.py

```diff
@@ -835,6 +848,10 @@ def event_loop_overlap(self):
             batch = self.get_next_batch_to_run()
             self.cur_batch = batch
 
+            if batch:
```

Comment:
There is a `if batch` condition right after this, should you put this block under L855?